content/FIDO2/FIDO2_WebAuthn_Developer_Guide/WebAuthn_Server_Authentication.adoc

```diff
@@ -0,0 +1,49 @@
+== FIDO2 WebAuthn Server Authentication
+
+=== Initiate Authentication
+Authentication works much like registration, except less complex because of the fewer parameters and absence of authenticator attestation complexities.
+
+To initiate an authentication operation, call the RelyingParty startAssertion method. The main parameter is the username of the user to authenticate, but even this is optional. If the username is not set, then the allowCredentials parameter will not be set. This means the user must use a client-side-resident credential to authenticate; also known as first-factor authentication.
+
+The startAssertion method returns an AssertionRequest containing the username, if any, and a PublicKeyCredentialRequestOptions instance which can be serialized to JSON and passed as the publicKey argument to navigator.credentials.get(). Again, tore the AssertionRequest in temporary storage so it can be passed as an argument to the RelyingParty finishAssertion method.
```

Comment:
See my other comment about JSON (de)serialization.

Reply:
Pointed out base64 to byteArray conversion for the get() in c4b1e40.  I plan to work with Emil to synchronize this section with the library api later.